gen-linux.sh

```diff
@@ -1,23 +1,5 @@
 #!/usr/bin/env bash
 set -ex
-
-# CI uses pre-built Boost
-if [[ -z "${CI}" ]]; then
-	# Create build dir
-	mkdir -p external/boost-build
-	cd external/boost
-
-	# Non-Windows needs to build Boost.Build first
-	chmod +x tools/build/src/engine/build.sh
-	sh bootstrap.sh
-
-	# Build our Boost subset
-	./b2 --ignore-site-config -j5 --build-dir=../boost-build --stagedir=../boost-build stage
-	cd ../..
-fi
-
-# Generate project files
```

Comment:
I don't think this is the strat. The best solution is to just fix what's wrong with https://github.com/Vita3K/ext-boost I think. Especially since it also doesn't work on windows (tried last night :/).

Reply:
Yeah, I think we should fix `ext-boost` too. I think we should either include the full `boost` (lazy and fast) or write some scripts to read all C header files in `Vita3K` to pull all the required files from `boost` which will take quite a while.

Reply:
Let's update `Vita3K` after `ext-boost` is fixed. I was temping to remove `ext-boost` from `.gitmodule` a while ago.

Reply:
We updated `ext-boost` in this [PR](https://github.com/Vita3K/ext-boost/pull/2). Please check to see if they are all working correctly with the updated `ext-boost` (`ext-boost` reference has been updated).